package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "@dfinity/ic-js",
-  "version": "0.7.0",
+  "version": "0.8.0",
```

Comment:
Question, this is not necessarily the version of all packages, right?

Reply:
Correct. It's the version of the root and the one we use to tag the repo.